securedrop_client/app.py

```diff
@@ -197,7 +198,7 @@ def start_app(args, qt_args) -> None:
     logging.info("Starting SecureDrop Client {}".format(__version__))
 
     app = QApplication(qt_args)
-    app.setApplicationName("SecureDrop Client")
+    app.setApplicationName(_("SecureDrop Client"))
```

Comment:
We have in the past suggested not translating/transliterating product names like "SecureDrop". I think that would hold here.

Reply:
oh, good catch @rmol!